site/src/main/java/uk/nhs/digital/common/enums/SearchArea.java

```diff
@@ -4,10 +4,12 @@
     ALL,
     DATA,
     SERVICES,
-    NEWS;
+    NEWS,
```

Comment:
Do we really want tabs for General and Hubs? We have previously decided with Owain to only have the tabs for All, Data and Information, Services and News and not to add any new ones.

Reply:
good point, this is part of the next US https://jira.digital.nhs.uk/browse/DW-223. Does it make sense to push that fix on a different branch/PR ?

Reply:
Would probably be easier to do it now as you'll end up changing exactly the same code here but it's up to you, I don't really mind 🙂